zerver/views/registration.py

```diff
@@ -701,7 +701,16 @@ def create_realm(request: HttpRequest, creation_key: Optional[str] = None) -> Ht
     return TemplateResponse(
         request,
         "zerver/create_realm.html",
-        context={"form": form, "current_url": request.get_full_path},
+        context={
+            "form": form,
+            "current_url": request.get_full_path,
+            "MAX_REALM_NAME_LENGTH": str(Realm.MAX_REALM_NAME_LENGTH),
+            "MAX_REALM_SUBDOMAIN_LENGTH": str(Realm.MAX_REALM_SUBDOMAIN_LENGTH),
+            "root_domain_available": is_root_domain_available(),
+            "sorted_realm_types": sorted(
+                Realm.ORG_TYPES.values(), key=lambda d: d["display_order"]
+            ),
```

Comment:
Similar question here -- should we have a function `get_realm_create_form_context` to do the most of this that should be shared?